client/v2/client_test.go

```diff
@@ -257,16 +258,19 @@ func TestClient_Concurrent_Use(t *testing.T) {
 	wg.Add(3)
 	n := 1000
 
+	errC := make(chan error, 3)
```

Comment:
Looks like you commented just as I realised I had some redundant loops in the test handling code. The pattern instead of `for` should just be `if err := <-errC; err != nil` since you only need to find one error